Manito/Manito/Screens/Detail-Ing/DetailngCodebaseViewController.swift

```diff
@@ -0,0 +1,354 @@
+        return label
+    }()
+    private lazy var statusLabel: UILabel = {
+        var label = UILabel()
+        label.backgroundColor = .mainRed
+        label.layer.masksToBounds = true
+        label.layer.cornerRadius = 11
+        label.textColor = .white
+        label.font = .font(.regular, ofSize: 13)
+        label.textAlignment = .center
+        return label
+    }()
+    private var missionBackgroundView: UIView = {
+        var view = UIView()
+        view.backgroundColor = .darkGrey004
+        view.layer.borderWidth = 1
+        view.layer.cornerRadius = 10
+        view.layer.borderColor = UIColor.systemYellow.cgColor
+        return view
+    }()
+    private let missionTitleLabel: UILabel = {
+        let label = UILabel()
+        label.text = "오늘의 개별 미션"
+        label.textColor = .grey002
+        label.font = .font(.regular, ofSize: 14)
+        return label
+    }()
+    private lazy var missionContentsLabel: UILabel = {
+        var label = UILabel()
+        label.textColor = .white
+        label.font = .font(.regular, ofSize: 18)
+        return label
+    }()
+    private let informationTitleLabel: UILabel = {
+        let label = UILabel()
+        label.text = "진행 정보"
+        label.textColor = .white
+        label.font = .font(.regular, ofSize: 16)
+        return label
+    }()
+    private lazy var manitteeBackView: UIView = {
+        let view = UIView()
+        let tapGesture = UITapGestureRecognizer(target: self, action: #selector(didTappedManittee))
+        view.addGestureRecognizer(tapGesture)
+        view.backgroundColor = .darkGrey002
+        view.makeBorderLayer(color: .white)
+        return view
+    }()
+    private let manitteeImageView: UIView = {
+        let view = UIView()
+        view.backgroundColor = .subOrange
+        view.layer.cornerRadius = 49.5
+        return view
+    }()
+    private var manitteeIconView: UIImageView = {
+        var imageView = UIImageView()
+        imageView.image = ImageLiterals.icManiTti
+        return imageView
+    }()
+    private lazy var manitteeLabel: UILabel = {
+        var label = UILabel()
+        label.textColor = .white
+        label.font = .font(.regular, ofSize: 15)
+        return label
+    }()
+    private lazy var listBackView: UIView = {
+        let view = UIView()
+        let tapGesture = UITapGestureRecognizer(target: self, action: #selector(pushFriendListViewController(_:)))
+        view.addGestureRecognizer(tapGesture)
+        view.backgroundColor = .darkGrey002
+        view.makeBorderLayer(color: .white)
+        return view
+    }()
+    private let listImageView: UIView = {
+        let view = UIView()
+        view.backgroundColor = .subPink
+        view.layer.cornerRadius = 49.5
+        return view
+    }()
+    private let listIconView: UIImageView = {
+        let imageView = UIImageView()
+        imageView.image = ImageLiterals.icList
+        return imageView
+    }()
```

Comment:
p5;

```suggestion
    private let listIconView = UIImageView(image: ImageLiterals.icList)
```

이렇게 한 줄로 쓸 수 있을 거 같네요

Reply:
> p5;
> 
> 이렇게 한 줄로 쓸 수 있을 거 같네요

감사합니다~ a77ba33

Reply:
p3;

클로저 안에 있는 `imageView`말고 listIconView에다가 요렇게 써주세요!
```swift
private let listIconView = UIImageView(image: ImageLiterals.icList)
```

Reply:
> p3;
> 
> 클로저 안에 있는 `imageView`말고 listIconView에다가 요렇게 써주세요!
> 
> ```swift
> private let listIconView = UIImageView(image: ImageLiterals.icList)
> ```

4de6d93 감사합니다!